pkg/blobinfocache/boltdb/boltdb_test.go

```diff
@@ -1,30 +1,31 @@
-package blobinfocache
+package boltdb
 
 import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"testing"
 
+	"github.com/containers/image/pkg/blobinfocache/internal/test"
 	"github.com/containers/image/types"
 	"github.com/stretchr/testify/require"
 )
 
-func newTestBoltDBCache(t *testing.T) (types.BlobInfoCache, func(t *testing.T)) {
+func newTest(t *testing.T) (types.BlobInfoCache, func(t *testing.T)) {
```

Comment:
Rename this to `newCache` or `newTestCache`, please (or keep the original name).

Reply:
>     Rename this to `newCache` or `newTestCache`

`newTestCache` with b2f8be9 -> d545a19.